readme.txt

```diff
@@ -195,6 +195,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= TBD
+
+* Tweak - Updated sharedCapacity prop type to number. This avoids warnings in jest testing [ET-1058]
```

Comment:
Since the changelog is public, I don't think we need to mention anything about internal tests.
```suggestion
* Tweak - Updated sharedCapacity prop type to number within the tickets block to avoid warning logs. [ET-1058]
```

Reply:
Thanks @codingmusician. Suggestion applied